gradle-plugins/src/main/java/com/linkedin/pegasus/gradle/PegasusPlugin.java

```diff
@@ -1710,6 +1696,11 @@ protected GenerateDataTemplateTask configureDataTemplateGeneration(Project proje
               copySpec.eachFile(fileCopyDetails ->
                   fileCopyDetails.setPath(TRANSLATED_SCHEMAS_DIR + File.separatorChar + fileCopyDetails.getPath())));
 
+          // Copy all extension schemas as-is into the root extensions directory in the JAR
+          task.from(publishableExtensionSchemasBuildDir, copySpec ->
```

Comment:
in gradle, copy and sync task are alike except sync will remove existing files in the destination dir.
I think maybe in this we should use sync task instead of copy to remove existing data template to keep the destination dir clean?

Reply:
I thought about removing schemas from destination dir. However, I think for debugging purpose, we should keep the schema in the build dir. So that we clearly know what has been packed in the data template jar.